apex_legends/base.py

```diff
@@ -44,3 +64,35 @@ def request(self, endpoint):
                 response.status_code, Exception)
             raise exception
         return json.loads(response.text)
+
+
+class AsyncClient:
```

Comment:
Maybe you could extend from client so you don't need to redefine the base url and de api errors codes.

Reply:
Also, could you please use 4 spaces instead of 2? :smiley:

Reply:
I attempted to subclass the original `Client` already. The problem with this is that the `requests` session that's created in the regular Client would be brought into the AsyncClient and never be used. This can still be done, I just don't wish to have that dragged over.

Reply:
Also, I can push out the indentation changes later today.
Blame REPL.it for that.

Reply:
> I attempted to subclass the original `Client` already. The problem with this is that the `requests` session that's created in the regular Client would be brought into the AsyncClient and never be used. This can still be done, I just don't wish to have that dragged over.

If you override the init method it shouldn't create the _requests_ session.